photons_interactor/commander/commands/control.py

```diff
@@ -94,7 +93,7 @@ class TransformCommand(store.Command):
 
     async def execute(self):
         fltr = chp.filter_from_matcher(self.matcher, self.refresh)
-        msg = Transformer.using(self.transform)
+        msg = Transformer.using(self.transform, **self.transform_options)
```

Comment:
This generates the following error for me :`AttributeError: 'TransformCommand' object has no attribute 'transform_options'`. I thought I'd updated photons-core in the venv properly, but I pushed it anway, so you can take a look too.